app.rb

```diff
@@ -109,11 +109,13 @@ class InvalidChannelName < StandardError; end
   end
 
   get '/chef/install.msi' do
-    redirect to('/stable/chef/download?p=windows&pv=2008r2&m=x86_64')
+    # default to 32-bit architecture for now
+    redirect to('/stable/chef/download?p=windows&pv=2008r2&m=i686')
```

Comment:
Even though `i686` works here we should use `i386` as this is the standard 32-bit architecture indicator we use across our release infra. See the following for more details:
https://github.com/chef/omnitruck/blob/c764a5974bec01fbd825cfdc1b15124e8e5f928e/app.rb#L265